src/pymorize/core/validate.py

```diff
@@ -187,8 +187,7 @@ class RuleSectionValidator(DirectoryAwareValidator):
                     "schema": {"type": "string"},
                 },
                 "cmor_units": {"type": "string", "required": False},
-                # FIXME(PS): How is it currently defined?
-                "model_units": {"type": "string", "required": False},
+                "model_unit": {"type": "string", "required": False},
```

Comment:
Since this is a string and refers to the unit of a value, it's okay to keep it singular IMO (`model_unit`) but then I would also make `cmor_units` singular (`cmor_unit`) to avoid confusion. That will involve some `grep cmor_unit`/`sed` in the code and documentation to fix it.

Reply:
What happens in the case when you need to combine 1+n number of variables for your one output? These could have different units for each variable of each file...

I would suggest that `"model_unit"` basically just refers to the source unit of the `DataArray` for the transformation of units.